src/main/java/by/gexateq/simplerestservice/controller/EmployeeController.java

```diff
@@ -0,0 +1,48 @@
+package by.gexateq.simplerestservice.controller;
+
+import by.gexateq.simplerestservice.entity.Employee;
+import by.gexateq.simplerestservice.service.EmployeeService;
+import lombok.AllArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Optional;
+
+@RestController
+@AllArgsConstructor
+@RequestMapping("/api")
+public class EmployeeController {
+    private final EmployeeService employeeService;
+
+    @PostMapping(value = "/create")
```

Comment:
/create in URL violates REST concept. We should represent the resource, not action. The resource is employee. The actions should be represented via Http methods (get, post, etc) Same for other endpoints.